### PR TITLE
haskell: disable runghc strip by default.

### PR DESCRIPTION
**Pull request by eric-ch**
runghc will strip binaries by default. This is not expected by
bitbake/OE.

### CODE REVIEW

File: recipes-devtools/hackage/hkg-vhd_0.2.bb [hunk 1]
```diff
@@ -14,7 +14,3 @@ DEPENDS += " \
     hkg-cereal \
     hkg-text \
 "
-
-INSANE_SKIP_${PN} = "already-stripped"
-
-PR = "r1"
```
Comment (jean-edouard) [inline recipes-devtools/hackage/hkg-vhd_0.2.bb:20]: Why remove it here but not in hkg-haxml?

File: recipes-devtools/hackage/hkg-haxml_1.20.2.bb [hunk 2]
```diff
@@ -9,7 +9,6 @@ SRC_URI[md5sum] = "9635c348e70c0446e74783e7c267050c"
 SRC_URI[sha256sum] = "c32c10b95446ecb938dc6cd34585187efd3fcb4b21f7d0c7cbd646ba94c87516"
 DEPENDS += "hkg-polyparse"
 
-INSANE_SKIP_${PN} += "already-stripped"
 INHIBIT_SYSROOT_STRIP = "1"
```
Comment (jean-edouard) [inline recipes-devtools/hackage/hkg-haxml_1.20.2.bb]: This line should probably go too, no?

**Review by jean-edouard — COMMENTED** [review]
Great fix, just a couple concerns.

**Comment by jean-edouard** [comment]
Built here: http://openxt-builder.ainfosec.com:8010/builders/openxt/builds/1259

File: recipes-devtools/hackage/hkg-haxml_1.20.2.bb (hunk 2 above)
Comment (eric-ch) [reply recipes-devtools/hackage/hkg-haxml_1.20.2.bb]: True, I missed that one.

**Comment by jean-edouard** [comment]
LGTM, merging soon.

**Pull request merged**